src/services/expenses.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+let expanses = [];
+let idsExpansesCounter = 0;
+
+function initialExpanses() {
```

Comment:
This name doesn't show what this function does, name it `setDefaultExpanses` or `clearDB` etc